src/modules/assets/assets-getter.service.ts

```diff
@@ -80,7 +80,7 @@ export class AssetsGetterService {
 
     if (filters.ownerAddress && filters.customFilters) {
       const [ticketsCollections] = await this.featuredCollectionsService.getFeaturedCollections(new FeaturedCollectionsFilter({ type: FeaturedCollectionTypeEnum.Tickets }))
-      if (ticketsCollections.length <= 0) return new CollectionType<Asset>()
+      if (ticketsCollections?.length === 0) return new CollectionType<Asset>()
```

Comment:
if `ticketsCollections` is `undefined` or `null`, then it will skip this `if` and throw an error on line 85 saying `cannot read properties of undefined (reading 'map')`

Reply:
done